pkgs/applications/misc/double-commander/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, stdenv
+, autoPatchelfHook
+, fetchFromGitHub
+, lazarus
+, fpc
+, libX11
+, glib
+, dbus
+
+  # GTK2
+, gtk2
+, atk
+, pango
+, gdk-pixbuf
+, cairo
+
+  # Qt5
+, libqt5pas
+, qt5
+, widgetset ? "qt5"
+}:
+
+stdenv.mkDerivation rec {
+  pname = "doublecmd";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "doublecmd";
+    repo = "doublecmd";
+    rev = "v${version}";
+    sha256 = "sha256-nX5EZekYF9saM0QaN9caCnsRDIqPYNX/BNu6Bb5lDYQ=";
```

Comment:
```suggestion
    sha256 = "sha256-DVEFflk+gEcqWbAvCri4v1O3oOOXUMBYFKRifDpX1y4=";
```

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/153228/commits/600a14b34a9c339c283a82ad585bc79a01596f95